std/math.d

```diff
@@ -3724,18 +3750,18 @@ real log10(real x) @safe pure nothrow @nogc
     assert(fabs(log10(1000) - 3) < .000001);
 }
 
-/******************************************
- *      Calculates the natural logarithm of 1 + x.
+/**
+ * Calculates the natural logarithm of 1 + x.
  *
- *      For very small x, log1p(x) will be more accurate than
- *      log(1 + x).
+ * For very small x, log1p(x) will be more accurate than
+ * log(1 + x).
  *
  *  $(TABLE_SV
  *  $(TR $(TH x)            $(TH log1p(x))     $(TH divide by 0?) $(TH invalid?))
  *  $(TR $(TD $(PLUSMN)0.0) $(TD $(PLUSMN)0.0) $(TD no)           $(TD no))
  *  $(TR $(TD -1.0)         $(TD -$(INFIN))    $(TD yes)          $(TD no))
- *  $(TR $(TD $(LT)-1.0)    $(TD $(NAN))       $(TD no)           $(TD yes))
- *  $(TR $(TD +$(INFIN))    $(TD -$(INFIN))    $(TD no)           $(TD no))
+ *  $(TR $(TD $(LT)-1.0)    $(TD -$(NAN))      $(TD no)           $(TD yes))
```

Comment:
Could you add an explicit test for this? (i.e. that it's a negative nan)

Reply:
I have a test for this on line 3798, but because `-real.nan != -real.nan` there's really no way to test this

Reply:
Use `is`.